R/imputation.R

```diff
@@ -26,7 +26,13 @@ setMethod("impute", "MSnSet",
               ## imputaton methods
               ##
               if (method == "knn") {
-                  .eset <- impute.knn(exprs(object), ...)$data
+                  imp_res <- impute.knn(exprs(object), ...)
+                  .eset <- imp_res$data
+                  if(!is.null(imp_res$rng.state)) {
+                   assign(".Random.seed", imp_res$rng.state, envir=.GlobalEnv)
```

Comment:
Here are 3 spaces missing (I guess it was my fault, using the web interface instead my editor ...):
```suggestion
                      assign(".Random.seed", imp_res$rng.state, envir=.GlobalEnv)
```